### PR TITLE
docs: add zeabur to web hosting

### PR DESCRIPTION
**Pull request by MichaelYuhe**
## Requirements

 * [x] This is Software as a Service not self hosted
 * [x] It has a free tier not just a free trial
 * [x] Pricing information is clearly visible without signup or phone calls
 * [x] The submission mentions what is free
 * [x] The submission is not already present in the list
 * [x] The service has contact details of those running it and a privacy policy

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1195,6 +1195,7 @@ Update Time, five active automations, webhooks.
   * [MDB GO](https://mdbgo.com/) - Free hosting for one project with two weeks Container TTL, 500 MB RAM per project, SFTP - 1G disk space.
   * [Patr Cloud](https://patr.cloud/) — An easy-to-use cloud platform, among its paid services it offers to host three static sites for free.
   * [Serv00.com](https://serv00.com/) — 3 GB of free web hosting with daily backups (7 days). Support: Crontab jobs, SSH access, repositories (GIT, SVN, and Mercurial), support: MySQL, PostgreSQL, MongoDB, PHP, Node.js, Python, Ruby, Java, Perl, TCL/TK, Lua, Erlang, Rust, Pascal, C, C++, D, R, and many more.
+  * [Zeabur](zeabur.com) - Deploy instantly and scale infinitely. Free for hosting serverless functions and static sites, free SSL and sub domain.
```
Comment (ripienaar) [inline README.md:1198]: It says $0+resource usage.  So that doesnt sound to me like its a free tier.

Comment (MichaelYuhe) [reply README.md:1198]: For serverless functions it's free to use

Comment (ripienaar) [reply README.md:1198]: It might be, but thats not what the site says in clear terms.

**Comment by madhuryadutta** [comment]
@ripienaar , This is a **duplicate** entry. **Zeabur**(zeabur.com) is already listed on **PaaS** section of the list.
I think you can close this pull request.

Thank you
Madhurya Dutta

image

**Comment by ripienaar** [comment]
Thanks @madhuryadutta

**Pull request closed without merging**